src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
```

Comment:
The border color is set to `$white-color`. Ensure that `$white-color` is defined in your variables file. If it's not defined, this will cause an error.